File: Programmers/Lv.1/자릿수더하기.py
def solution(n):
    answer = 0
    
    for i in str(n):
        answer += int(i)

    return answer

# 재귀
# def solution(n):
#     if n < 10:
#         return n
#     return (n % 10) + solution(n // 10)

File: Programmers/Lv.1.Kotlin/자릿수더하기.kt
class Solution {
    fun solution(n: Int): Int {
        return n.toString().map {
            it.toInt() - '0'.toInt()
        }.sum()
    }

    // fun solution(n: Int): Int {
    //     var input = n
    //     var answer = 0

    //     while (input != 0) {
    //         answer += input % 10
    //         input /= 10
    //     }

    //     return answer
    // }
}